docs/src/main/docs/tracing/02_Zipkin.adoc

```diff
@@ -0,0 +1,86 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Zipkin Tracing
+:description: Helidon Tracing Support
+= :keywords: helidon, tracing
+
+== Zipkin Tracing
+Helidon SE is integrated with Zipkin tracer.
+
+The Zipkin builder is loaded through `ServiceLoader` and configured. You could
+also use the Zipkin builder directly, though this would create a source-code dependency
+on Zipkin tracer.
+
+
+=== Maven Coordinates
+To use Zipkin as the collector of traces (e.g. a Tracer),
+    add the following dependency to your project:
+
+[source,xml,subs="verbatim,attributes"]
+.Zipkin Tracer
+----
+<dependency>
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+----
+
+=== Configuring Zipkin
+
+The following configuration options are supported by Zipkin tracer:
```

Comment:
... by the Zipkin tracer:

Better still, change to active voice: "The Zipkin tracer supports the following configuration options:"